ricecooker/utils/browser.py

```diff
@@ -21,8 +75,32 @@ def translate_path(self, path):
                 path = os.path.join(path, word)
             return path
 
-    httpd = HTTPServer(("127.0.0.1", port), RequestHandler)
+    httpd = HTTPServer((SERVER, PORT), RequestHandler)
+
+    webbrowser.open("http://{}:{}/{}".format(SERVER, PORT, filename))
+
+    httpd.serve_forever()
+
+def iframe_preview(filepath):
+
+    filename = "index.html"
+    directory = filepath
+    if not os.path.isdir(filepath):
+        filename = os.path.basename(filepath)
+        directory = os.path.dirname(filepath)
+
+    os.chdir(directory)
+    httpd = HTTPServer((SERVER, PORT), IFrameServerRequestHandler)
+
+    webbrowser.open("http://{}:{}/{}".format(SERVER, PORT, filename))
+
+    httpd.serve_forever()
 
-    webbrowser.open("http://127.0.0.1:{}/{}".format(port, filename))
+def load_server(path, iframe=False):
+    if iframe:
+        print("Starting local server and iframe sandbox...")
+        iframe_preview(path)
```

Comment:
I missed where the wrapper page gets served up, with the iframe itself, and its sandbox attribute?

Reply:
Right now the path is to the iframe itself, not the embedded content. Currently, to use this for testing, the code in the iframe itself will have to change depending on the test. (e.g. to check that the iframe parent received the messages for the test in the right order, etc.) See the packages/hashi/test directory on the https://github.com/learningequality/kolibri/tree/html5_app_bridge_library branch for a couple examples.